dmsite/venv/src/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -35,13 +95,79 @@ class SearchResultsPage extends React.Component {
     return searchResults.map(result => (
       <ListGroupItem>
         <SearchResultsCell dataset={result} history={this.props.history} />
-      </ListGroupItem>));
+      </ListGroupItem>
+    ));
+  }
+
+  renderFilterTags() {
+    return this.state.filterTags.map((tag, index) => (
+      <Button id="filterTag" color="secondary" size="sm" style={{ marginLeft: '.25rem' }} onClick={(index) => { this.removeFilterTag(index); }}>
+        {`\u2A09 ${tag.type}: ${tag.value}`}
```

Comment:
Might want to leave a comment here to indicate what this is, since this is literally nonsense.